### PR TITLE
[docs] [quick-start] add macOS monterey note to fix the 7000 port in use.

### LINKED ISSUE #11062: [docs] macOS startup workaround for port(s) in use
Slack thread: https://yugabyte.slack.com/archives/CNC26JH3P/p1641928603033600

macOS Monterey turns on AirPlay receiving by default, which listens on port 7000. This conflicts with YugabyteDB and causes `yugabyted start` to partially fail. The workaround is to turn AirPlay receiving off, then start YugabyteDB, and then (optionally) turn AirPlay receiving back on.

Add a note to this effect in the Quick Start, and perhaps also in Troubleshooting.

### PR DESCRIPTION
**Pull request by aishwarya24**
Resolves #11062 

- Added a note for the macOS quickstart page and in troubleshooting. 
- Changes are in latest currently. Will include them to stable once this is approved.

@netlify /latest/troubleshoot/nodes/check-processes/

### CODE REVIEW

**Review by vkulichenko — CHANGES_REQUESTED** [review]
Not every user will want to disable the AirPlay. We should also provide an alternative option of changing the port number for the UI. Here is the command to customize the port:

`./bin/yugabyted start --master_webserver_port=9999`

**Comment by aishwarya24** [comment]
@vkulichenko @polarweasel @dmagda , I have added the suggestion mentioned by @vkulichenko as well. This is probably the same as @dmagda mentioned earlier in the [issue](https://github.com/yugabyte/yugabyte-db/issues/11062).
Please review.

File: docs/content/latest/troubleshoot/nodes/check-processes.md [hunk 1]
```diff
@@ -73,3 +73,19 @@ sudo firewall-cmd --zone=public --add-port=9300/tcp;
...
+
+- macOS Monterey turns on AirPlay receiving by default, which listens on port 7000. This conflicts with YugabyteDB and causes `yugabyted start` to fail.
+
+```output
+$ ./bin/yugabyted start
+Starting yugabyted...
+/ Running system checks...Failed to bind to address:  0.0.0.0:7000
+```
+
+The workaround is to turn AirPlay receiving off, then start YugabyteDB, and then (optionally) turn AirPlay receiving back on.Alternatively(recommended), you can change the default port number using the `--master_webserver_port` flag when you start the cluster as follows:
```
Comment (polarweasel) [inline docs/content/latest/troubleshoot/nodes/check-processes.md:87]:
```suggestion
The workaround is to turn AirPlay receiving off, then start YugabyteDB, and then (optionally) turn AirPlay receiving back on. Alternatively(recommended), you can change the default port number using the `--master_webserver_port` flag when you start the cluster as follows:
```

**Review by polarweasel — APPROVED** [review]
LGTM!

**Pull request merged**